app/ide-desktop/lib/dashboard/src/components/AriaComponents/index.ts

```diff
@@ -2,7 +2,9 @@
  * @file index.ts
  * Index file for Aria Components
  */
-export * from './Button/Button'
+export * from './Button'
 export * from './Tooltip/Tooltip'
```

Comment:
is there a reason we deduplicated `Button`'s path but not `Tooltip`'s?

Reply:
mostly because the tooltip doesn't have a barrel import and Button now has(because it exposes now more that just Button, also - ButtonGroup, CopyButton, CloseButton)